freepl_main/.env.example

```diff
@@ -0,0 +1,2 @@
+SECRET_KEY=ajfbejqbt4j2b5jb4tkwjbkgjtbewkgjb
+DEBUG=True
```

Comment:
```
SOCIAL_AUTH_FACEBOOK_KEY 
SOCIAL_AUTH_FACEBOOK_SECRET 
SOCIAL_AUTH_GOOGLE_OAUTH2_KEY
SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET
```

Add these variables to .env and set them with decouple in `settings.py.example`

Reply:
done